quad_control/setup.py

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
The `setup.py` file is used by catkin to make python packages out of our folders.
If a folder is a python package, it can be imported by other python scripts, no matter what the relative position is.
So now, the folder `utilities` is a python package, and it can be imported by other scripts (for example `trajectory.py`) even if the folder `utilities` is not in the same folder as the file `trajectory.py`.